dali/pipeline/operator/op_schema.h

```diff
@@ -317,11 +317,26 @@ graph even if its outputs are not used.)code", false);
   }
 
   /**
-   * @brief Notes that this operator is deprecated and optionally specifies the operator to be used instead
+   * @brief Notes that for this operator only the doc_str should be visible, but not the docs for
+   * the inputs, outputs or argument (the Op is exposed in Python API)
    */
-  DLL_PUBLIC inline OpSchema& Deprecate(const std::string &in_favor_of) {
+  DLL_PUBLIC inline OpSchema &MakeDocPartiallyHidden() {
+    is_doc_partially_hidden_ = true;
+    return *this;
+  }
+
+  /**
+   * @brief Notes that this operator is deprecated and optionally specifies the operator to be used
+   * instead
+   *
+   * @param in_favor_of schema name of the replacement
+   * @param explanation additional explanation
+   */
+  DLL_PUBLIC inline OpSchema &Deprecate(const std::string &in_favor_of,
+                                        const std::string &explanation = "") {
     is_deprecated_ = true;
     deprecated_in_favor_of_ = in_favor_of;
+    deprecated_explanation_ = explanation;
```

Comment:
```suggestion
    deprecated_because_ = explanation;
```
or (but it sounds like cheap rap lyrics)
```suggestion
    deprecation_explanation_ = explanation;
```
or
```suggestion
    deprecation_message_ = explanation;
```

Reply:
done